data/styles/default/include/priorities_4_overlays.prio.txt

```diff
@@ -748,6 +748,7 @@ amenity-bank                                        # icon z16- (also has captio
 amenity-library                                     # icon z18- (also has caption(optional) z18-)
 amenity-marketplace                                 # icon z16- (also has caption(optional) z16-)
 amenity-pharmacy                                    # icon z16- (also has caption(optional) z17-)
+shop-hearing_aids                                   # icon z16- (also has caption(optional) z16-)
```

Comment:
this is a highly specialized / niche shop, please put it at the lower priority 500